imports/plugins/core/versions/server/migrations/25_deafult_parcel_size.js

```diff
@@ -0,0 +1,25 @@
+import { Migrations } from "meteor/percolate:migrations";
+import { Shops } from "/lib/collections";
+import { convertWeight, convertLength } from "/lib/api";
+
+Migrations.add({
+  version: 25,
+  up() {
+    const shops = Shops.find().fetch();
+    shops.forEach((shop) => {
+      const defaultParcelSize = {
+        weight: convertWeight("lb", shop.baseUOM, 8),
+        height: convertLength("in", shop.baseUOL, 6),
+        length: convertLength("in", shop.baseUOL, 11.25),
+        width: convertLength("in", shop.baseUOL, 8.75)
+      };
+      Shops.update({ _id: shop._id }, { $set: { defaultParcelSize } });
+    });
+  },
+  down() {
+    const shops = Shops.find().fetch();
+    shops.forEach((shop) => {
+      Shops.update({ _id: shop._id }, { $unset: { defaultParcelSize: 1 } });
```

Comment:
Add a third argument: `{ bypassCollection2: true }`

Also, this one can be done as a multi update (remove the `find` and `forEach`).

```js
Shops.update({}, { $unset: { defaultParcelSize: 1 } }, { bypassCollection2: true, multi: true });
```